CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.13.0-rc.0 / 2019-09-25
+
+* [CHANGE] Metrics: Change prometheus_sd_configs_failed_total to Gauge #5254
+* [CHANGE] Service discovery: add new node address types for kubernetes. #5902
+* [CHANGE] Metrics format: update parsing to disallow new Go 1.13 float formats. #5984
+* [ENHANCEMENT] Remote read: use remote read streaming to reduce memory usage. #5703
+* [ENHANCEMENT] Docs: fix spelling mistakes and other small changes . #5952, #6051
```

Comment:
i don't think we list those changes usually.

Reply:
yea probably right, bigger docs changes would be good to list, but these small ones are not helpful.

Reply:
Even big doc changes don't get listed, as they don't affect the binary

Reply:
removed